### ISSUE TITLE
Request: add support for deflate encoding

### ISSUE THREAD

**Issue by aharpervc** [danielgtaylor/restish#205]
Hi, I've started playing with restish, very cool project!

My request is to add support for `deflate` encoding. I see by default the accept-encoding header is `br,gzip` but I'm working with an API that only supports deflate.

I confirmed that I can override that header with a persistent header in my config file, but that results in an error: `Caught error: unsupported content-encoding deflate`

**Comment by aharpervc** [comment]
Hi, thanks for adding this feature. I've now been able to revisit restish since you added it. I noticed that with my API, the response body is no longer printed when the response is deflate encoded. I can add `--rsh-header 'accept-encoding:identity'` and the response body is printed as normal. With `--rsh-verbose`, I see various ascii codes dumped out with <?>'s, I'm guessing that's the raw response, and `DEBUG: Decoding response from deflate`. No errors are indicated.

Is this expected? Can restish be configured to show the response body after decompression?
